piserver/app.py

```diff
@@ -50,7 +64,52 @@
             ms2_pin=MS2_PIN,
             enable_pin=ENABLE_PIN)
 
-#TODO: Init a motor handler here
+# setup for auth with jwt
+if not "TOKEN_DURATION_MINUTES" in app.config.keys():
+    app.config[ "TOKEN_DURATION_MINUTES" ] = 10
+else:
+    app.config[ "TOKEN_DURATION_MINUTES" ] = int( app.config[ "TOKEN_DURATION_MINUTES" ] )
+if not "PISERVER_SECRET_KEY" in app.config.keys():
+    app.config[ "PISERVER_SECRET_KEY" ] = "willekeurigegeheimesleutel"
+else:
+    app.config[ "PISERVER_SECRET_KEY" ] = str( app.config[ "PISERVER_SECRET_KEY" ] )
+
+'''
+Small database model for the users.
+Used for authentication with JSON web tokens
+'''
+class User( db.Model ):
+    id = db.Column( db.Integer, primary_key=True )
+    public_id = db.Column( db.String(50), unique=True )
+    name = db.Column( db.String(30), unique=True )
+    password = db.Column( db.String(50) )
+
+'''
+Decorator for using the JWT token
+'''
+def token_required( fxn ):
```

Comment:
Are we able to check the ip the request comes from and if it's coming from the same machine as the server, just ignore auth?

Reply:
i think it's hard when the host is set to like 0.0.0.0 or something like that. I also thought that we'd sometimes want to test the auth locally anyway. I can look into it if you'd like.